02/app.js

```diff
@@ -8,4 +8,8 @@ const calendarJS = {
     'ES7': '2016-06',
     'ES8': '2017-06',
     'ES9': '2018-06',
+}
+for(const key in calendarJS) {
+    if(calendarJS[key] == null) console.log(key, "nie zostało wydane");
+    else console.log(key, "zostało wydne w terminie", calendarJS[key]);
```

Comment:
Raczej jestem zwolennikiem pisania "wąsów" tj. `{ ... }`, wtedy kod jest czytelniejszy i jak się pojawią nowe instrukcje to nie zapomnimy o napisania właśnie nawiasów :)